From: Colin Ian King <colin.king@canonical.com>
Date: Fri, 30 Jun 2017 11:59:22 +0100
Subject: net/mlx5: fix spelling mistake: "Allodating" -> "Allocating"
Patch-mainline: v4.13-rc1
Git-commit: 4120dab09574e29e7576895342896ccb21b336e7
References: bsc#1046303 FATE#322944

Trivial fix to spelling mistake in mlx5_core_dbg debug message

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Reviewed-by: Ilan Tayari <ilant@mellanox.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/net/ethernet/mellanox/mlx5/core/lib/gid.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/ethernet/mellanox/mlx5/core/lib/gid.c
+++ b/drivers/net/ethernet/mellanox/mlx5/core/lib/gid.c
@@ -101,7 +101,7 @@ int mlx5_core_reserved_gid_alloc(struct
 	if (index < 0)
 		return index;
 
-	mlx5_core_dbg(dev, "Allodating reserved GID %u\n", index);
+	mlx5_core_dbg(dev, "Allocating reserved GID %u\n", index);
 	*gid_index = index;
 	return 0;
 }
